File: ios/MessagingInfo.m
#import <Foundation/Foundation.h>
#import "React/RCTBridgeModule.h"
#import "React/RCTEventEmitter.h"

@interface RCT_EXTERN_MODULE(MessagingInfo, NSObject)

RCT_EXTERN_METHOD(getSentMessage:(NSString *)message
                  resolve:(RCTPromiseResolveBlock)resolve
                  reject:(RCTPromiseRejectBlock)reject)

@end

File: ios/MessagingInfo.swift
import Foundation
import React

@objc(MessagingInfo)
class MessagingInfo: NSObject {

  @objc func getSentMessage(_ message: String, resolve: @escaping RCTPromiseResolveBlock, reject: @escaping RCTPromiseRejectBlock) {
    let response = "Received: \(message)"
    resolve(["response": response])
  }
  
}
